web/teacher/src/pages/settings/mail.vue

```diff
@@ -27,14 +28,32 @@ export default defineComponent({
     TheTextField,
   },
 
-  setup() {
-    const form = {
+  setup(_, { root }: SetupContext) {
+    const router = root.$router
```

Comment:
他の実装参考にしてくれてのじっそうかもだけど、山ちゃんが言うように、この書き方、今は非推奨になってる🙇‍♂️

なので、下の書き方で揃えてもらえると🙏

```suggestion
    const router = useRouter()
```

Reply:
それで書いた方がいいのかなとは思ってたけど、そうやって買いたらrouter.pushって書き方はできないよね？

Reply:
`router.push`使えるよ👀

https://composition-api.nuxtjs.org/packages/routes/